src/helpers/getTabById.ts

```diff
@@ -0,0 +1,8 @@
+import { Tab } from '../types';
+
+export const getTabById = (
+  tabsList: Tab[],
+  id: string,
+) => (
+  tabsList.find(tab => tab.id === id)
+);
```

Comment:
let's handle `undefined` in this function instead of our component.
```suggestion
export const getTabById = (
  tabsList: Tab[],
  id: string,
) => (
  tabsList.find(tab => tab.id === id) || tabsList[0]
);
```